monsteer/plugin/spikeReport.cpp

```diff
@@ -49,19 +50,26 @@ namespace
 const uint32_t INTERNAL_TIMEOUT = 500;
 }
 
+URI toHostAndPort( const URI& uri )
+{
+    URI out;
+    out.setHost( uri.getHost( ));
+    out.setPort( uri.getPort( ));
+    return out;
+}
+
 SpikeReport::SpikeReport( const brion::SpikeReportInitData& pluginData )
-    : _uri( pluginData.getURI( ))
+    : _uri( toHostAndPort( pluginData.getURI( )))
```

Comment:
A good idea, now let me ask the following question: why zeq and not servus? It is a generic use case which should be supported in both, right? And then, you still leave to the user the burden of splitting a "hostname:port" string from a command line parameter, another common use case. Which brings me back to my proposition above: let sevus::URI do it with a static constructor...